test/e2e/spinup_test.go

```diff
@@ -213,15 +213,15 @@ func ruler(i int, rules string) (cmdScheduleFunc, string) {
 	}, rulerCluster(i)
 }
 
-func minio(accessKey string, secretKey string) cmdScheduleFunc {
+func minio(ctx context.Context, accessKey string, secretKey string) cmdScheduleFunc {
 	return func(workDir string, clusterPeerFlags []string) ([]*exec.Cmd, error) {
 		dbDir := fmt.Sprintf("%s/data/minio", workDir)
 
 		if err := os.MkdirAll(dbDir, 0777); err != nil {
 			return nil, errors.Wrap(err, "creating minio dir failed")
 		}
 
-		cmd := exec.Command(testutil.MinioBinary(),
+		cmd := exec.CommandContext(ctx, testutil.MinioBinary(),
```

Comment:
Let's remove this, since it is duplication of our `Process.Kill` (the logic is the same)